jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -257,6 +260,11 @@
 
 	private transient Class<?> cachedEnclosingClass;
 	private static long cachedEnclosingClassOffset = -1;
+	
+/*[IF JAVA_SPEC_VERSION >= 12]*/
+	private transient ReflectionFactory reflectionFactory;
```

Comment:
This should be static.

Reply:
Restored back to `static` as suggested below.